src/style.css

```diff
@@ -1,3 +1,72 @@
-body {
+* {
+  padding: 0;
   margin: 0;
+  box-sizing: border-box;
+}
+
+body,
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. In this case, if the 'Roboto' font fails to load, the browser will use the 'sans-serif' font. Refer to the checklist item: 'Remember to use fallback fonts - alternative font-family in case the main one doesn't work'.